### ISSUE TITLE
[BUG] Installation of Fritter beta versions, latest ci build from github fails

### ISSUE THREAD

**Issue by nannadroid** [jonjomckay/fritter#739]
**Describe the bug**
When I install beta and ci build versions from github, installation fails (message screenshot is attached).

I first tried installing beta9 apk files from [releases](https://github.com/jonjomckay/fritter/releases) page. But all of them say "App not installed".
Then, i tried according to this [suggestion](https://github.com/jonjomckay/fritter/pull/714#issuecomment-1560040011).
Still got installation failed message.

All this is to install fix for "long tweets getting cut off", bug #711.

Please help fix this.

**To Reproduce**
1. Go to [releases](https://github.com/jonjomckay/fritter/releases) 
2. Download appropriate apk, and install
3. You'll see error "App not installed"

**Screenshots**
Attached

**Device**
OnePlus6, Android 11, OxygenOS 11.2.2
Fritter: v3.0.0-beta6+300000772
![Screenshot_20230524-005825__01](https://github.com/jonjomckay/fritter/assets/113890947/705d462e-8e69-435f-abf2-2d8ac0b8e5cf)

**Comment by j-fbriere** [comment]
Have you change the setting to allow your device to install an apk file?
Reference: [OXYGENOS 11 : HOW TO INSTALL APPLICATION USING APK](https://www.gadgetdetail.com/guides/oxygenos-11-how-to-install-application-using-apk/)
You should allow the **My Files** application to install unknow apps.
Then, after downloading and unzipping the [apk](https://github.com/TheHCJ/Quacker/suites/13193086794/artifacts/718276476) file, you should try to install it from the the **My Files** application.

**Comment by nannadroid** [comment]
> Have you change the setting to allow your device to install an apk file? Reference: [OXYGENOS 11 : HOW TO INSTALL APPLICATION USING APK](https://www.gadgetdetail.com/guides/oxygenos-11-how-to-install-application-using-apk/) You should allow the **My Files** application (or any other file explorer application you're using) to install unknow apps. Then, after downloading and unzipping the [apk](https://github.com/TheHCJ/Quacker/suites/13193086794/artifacts/718276476) file, you should try to install it from the **My Files** application (or any other file explorer application you're using).

Yes, i have done. I use Solid explorer and have enabled "
![Screenshot_20230528-205705](https://github.com/jonjomckay/fritter/assets/113890947/f6d5f9c1-0688-47ea-88b1-cda9518f6501)
Install unknown apps".
Not sure if anything else is needed.

**Comment by nannadroid** [comment]
And, is there fritter installation log, that can help understand what is wrong?

**Comment by j-fbriere** [comment]
About the logs, unfortunately the only way to look at the installation log is to use logcat. 
To achieve that, the device must be connected to a computer where the Android SDK is installed.

Maybe you could try one thing: uninstall the old fritter before installing the new version of fritter with the latest [apk](https://github.com/jonjomckay/fritter/suites/13127697925/artifacts/713264733) file.

Important note: Before uninstalling fritter be sure to export your settings/subscriptions/groups, because those will be deleted when the app is uninstalled. After the installation of the new version is finished, you will then be able to import your settings/subscriptions/group back to the app.

**Comment by nannadroid** [comment]
Ok, I can try uninstalling and installing fresh apk. I have exported the data and confirmed that json file indeed has 'saved' tweets, up to today. I have lot of saved tweets. Does import work flawlessly? I remember seeing an issue raised regarding this.

This is the reason I haven't tried uninstalling until now 🙂

**Comment by j-fbriere** [comment]
I know that the export/import of settings/subscriptions/groups works well.
The only caveat is that the blue checks of the blue check users do not appear for the subscriptions list after the import (you can still see the blue check on the user's profile or in their timeline). It's a minor inconvenience I think.

I don't save tweets, so I have no experience with the export/import of saved tweets.

**Comment by nannadroid** [comment]
I tried importing in Quacker, fork of Fritter. Saved tweets weren't imported at all. So, i can't trust Fritter as well, unless someone confirms.
Seems there is no solution for my problem 😐

**Comment by nannadroid** [comment]
>The only caveat is that the blue checks of the blue check users do not appear for the subscriptions list after the import (you can still see the blue check on the user's profile or in their timeline). It's a minor inconvenience I think.
>

Interesting!! In my version, blue ticks aren't visible anyways 😁

**Comment by j-fbriere** [comment]
Can you provide an exported json file with only one or two tweets in it, so that I can debug the problem?

**Comment by nannadroid** [comment]
sorry for the delayed response. I did some digging and found that it was my fault all along!!
1. about importing saved tweets: it works in both Fritter and Quacker. After importing, i was seeing the saved tab immediately. This doesn't work. After I closed the app and then open, I could see all the tweets.
2. with this, i come to main issue of installation fail: i uninstalled Fritter and installed latest version (beta9), now it got installed.

So, all issues solved. Big thanks for patient reading and support!
